### DISCUSSION TITLE
how to bring plotter to front

### DISCUSSION THREAD

**Discussion by oliverlee** [pyvista/pyvista#6652] [Q&A]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
import numpy as np
import pyvista as pv

points = np.random.random((100, 3))
pl = pv.plot(points)
```
```


### Description

Is it possible to configure the plot window to be active and show up in front of other windows?

I've only found this: https://discourse.vtk.org/t/setting-vtk-window-always-on-top/4947

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Sun Sep 15 09:43:40 2024 PDT

                OS : Darwin (macOS 14.6.1)
            CPU(s) : 8
           Machine : arm64
      Architecture : 64bit
       Environment : Python
        GPU Vendor : Apple
      GPU Renderer : Apple M1
       GPU Version : 4.1 Metal - 88.1
  MathText Support : True

  Python 3.12.4 (main, Jul 25 2024, 22:11:22) [Clang 18.1.8 ]

           pyvista : 0.44.1
               vtk : 9.3.1
             numpy : 2.1.1
        matplotlib : 3.9.2
            scooby : 0.10.0
             pooch : 1.8.2
            pillow : 10.4.0
--------------------------------------------------------------------------------
```

**Comment by banesullivan** [comment]
You could likely do this by embedding the plotter into a basic Qt application with pyvistaqt https://github.com/pyvista/pyvistaqt
